package.json

```diff
@@ -17,13 +17,11 @@
     "url": "git://github.com/modusintegration/mojaloop-payment-manager-management-api-service.git"
   },
   "scripts": {
-    "clean": "rimraf dist/",
-    "build": "npm run clean && tsc && npm run copy-files && rm dist/src/lib/cacheDatabase/migrations/*.ts",
-    "start": "npm run start:no-build",
-    "start:no-build": "node -r ./tsconfig-paths.js dist/src/index.js",
-    "start:build": "npm run build && npm run start:no-build",
-    "start:dev": "ts-node -r dotenv/config -r tsconfig-paths/register -P ./tsconfig.json src/index.ts",
-    "copy-files": "copyfiles -u 1 src/**/*.yaml dist/src",
+    "build": "tsup && npm run copy-files",
```

Comment:
Are we going to use this new "tsup" command to build? If yes, you may need to present this in our group about its advantages and the reason to migrate.

Reply:
It's just a bit more performant way to transpile TS to JS with some extra features like: generate JS for multiple module targets (e.g. CJS, ESM), bundling.  It allowed us inside _inter-scheme-proxy-cache-lib_ to generate different JS code to use it with any modules with ease:
![image](https://github.com/user-attachments/assets/ae0731f6-5015-4341-83df-aa98923e1df5)

https://blog.atomrc.dev/p/typescript-library-compilation-2023/

Reply:
No we don't have strict rule, but we defined a template repo for typescript projects with all the tools we want to use in Infitx. If there are any deviations in the major components like this, we need to discuss. You just need to discuss and convince the team, we are more than happy to try new tech.